cfme/tests/cli/test_appliance_log_rotate.py

```diff
@@ -0,0 +1,38 @@
+import datetime
+
+import dateutil
+
+
+def advance_appliance_date_by_day(appliance):
+    txt_date = appliance.ssh_client.run_command('date --rfc-3339=ns').output
+    appliance_date = dateutil.parser.parse(txt_date)
+    td = datetime.timedelta(days=1)
+    advanced_txt_date = (appliance_date + td).strftime("%Y-%m-%d %H:%M:%S%z")
+    appliance.ssh_client.run_command("date -s '{}'".format(advanced_txt_date))
+
+
+def log_rotate_check(configured_appliance, log_path):
+    configured_appliance.ssh_client.run_command("/etc/cron.daily/logrotate")
+
+    initial_log_files = configured_appliance.ssh_client.run_command("ls -1 {}*".format(
+        log_path)).output.split('\n')
+    configured_appliance.ssh_client.run_command(
+        "echo 'Ensure line in logs' >> {}".format(log_path))
+    advance_appliance_date_by_day(configured_appliance)
+
+    # Perform the logrotate.
+    configured_appliance.ssh_client.run_command("/etc/cron.daily/logrotate")
+
+    adv_time_log_files = configured_appliance.ssh_client.run_command("ls -1 {}*".format(
+        log_path)).output.split('\n')
+    assert set(initial_log_files) < set(adv_time_log_files)
+
+
+def test_appliance_vmdb_log_rotate(temp_appliance_preconfig_funcscope):
+    log = "/var/www/miq/vmdb/log/evm.log"
+    log_rotate_check(temp_appliance_preconfig_funcscope, log)
+
+
+def test_appliance_postgres_log_rotate(temp_appliance_preconfig_funcscope):
```

Comment:
So these two methods look to be the same thing. Have you thought about creating a global variable ```CFME_LOGS```, for example, which would be a dict of the different logs to rotate and paramertize the test of ```CFME_LOGS``` This will reduce the

Reply:
If we're only going to have these two tests, then it's maybe not necessary. However I definitely agree that what @jawatts proposes would be a better design. If you plan to add more tests in the future, I also propose to go that way.

Reply:
No. I don't plan to add any more tests.

Reply:
No. I don't plan to add any more tests.

Reply:
No. I don't plan to add any more tests.

Reply:
@jaryn I would still paramertize this, I do not think we should make this assumption that no more tests will be added and it reduce code duplications

Reply:
@jaryn please parameterize this.

Reply:
@jaryn Please paramertize this test. This is a required change.